src/java/org/apache/cassandra/cql3/statements/UpdateStatement.java

```diff
@@ -110,6 +124,17 @@ public void addUpdateForKey(PartitionUpdate.Builder update, Slice slice, UpdateP
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public ResultMessage execute(QueryState state, QueryOptions options, long queryStartNanoTime)
+    {
+        ResultMessage result = super.execute(state, options, queryStartNanoTime);
+
+        if (result == null) result = new ResultMessage.Void();
```

Comment:
Wanted a localized statement code change for now but need to bubble this code up to higher and unified abstractions (probably `ModificationStatement`) but there is no common implementation that cover both `Modification` and `SelectStatment` so probably some code duplication will remain.

Reply:
not valid anymore as I reduced the change to `SelectStatement`